tokenLists/fantom.mainnet.tokenlist.json

```diff
@@ -522,16 +512,15 @@
             "_scan": "https://ftmscan.com/token/0x0789fF5bA37f72ABC4D561D00648acaDC897b32d",
             "chainId": 250,
             "decimals": 18,
-            "logoURI": "https://assets.spookyswap.finance/tokens/MORPH.png"
+            "logoURI": "https://s2.coinmarketcap.com/static/img/coins/64x64/11896.png"
         },
         {
             "name": "Iron Finance ICE Token",
             "symbol": "IronICE",
             "address": "0x260b3E40c714Ce8196465Ec824Cd8Bb915081812",
             "_scan": "https://ftmscan.com/token/0x260b3E40c714Ce8196465Ec824Cd8Bb915081812",
             "chainId": 250,
-            "decimals": 18,
-            "logoURI": "https://assets.spookyswap.finance/tokens/IronICE.png"
+            "decimals": 18
```

Comment:
add logo

Reply:
The URL is no longer accessible and I couldn't find the logo URL in CoinMarketCap and ftmscan.com, so I removed it